lib/tasks/say.js

```diff
@@ -68,6 +68,12 @@ class TaskSay extends Task {
     const salt = cs.callSid;
 
     let credentials = cs.getSpeechCredentials(vendor, 'tts', label);
+
+    // override Synthesizer options from config verb
+    if (cs.synthesizer && cs.synthesizer.options) {
```

Comment:
@xquanluu  I would add these conditions: 
- call session synthesizer options are only applied, when there is no say.synthesizer.options
- at least one "key" must be present in call session synthesizer options 
```
if (Object.keys(this.options).length === 0 && cs.synthesizer && cs.synthesizer.options && Object.keys(cs.synthesizer.options).length > 0) {
```

Reply:
Thanks @Catharsis68 for the review:
1/ call session synthesizer options are only applied, when there is no say.synthesizer.options
--> This seem to me we violate the rule for config verb to override configuration from other verb.

2/ at least one "key" must be present in call session synthesizer options
--> Do you know any case that user use config verb to put {} empty configuration, IMHO I think there is 2 normal case there is  synthesizer.options and there is no synthesizer.options?

Reply:
I agree with @Catharsis68 on this one -- the config provides session-level settings, but if a verb specifies something different, that should take preference for that verb.  Given that, I think we can merge this @xquanluu ?